requirements.txt

```diff
@@ -6,6 +6,7 @@ lxml
 Pillow
 pycairo
 pyenchant
+pyglet
```

Comment:
Nice catch :) 
Could you also add a `python3-pyglet` entry in the [INSTALL.md](https://github.com/kirienko/gourmet/blob/master/INSTALL.md#unbutu-2004-linux-mint-20) ?